src/OrchardCore.Modules/OrchardCore.Layers/Deployment/AllLayersDeploymentSource.cs

```diff
@@ -15,28 +13,21 @@ public class AllLayersDeploymentSource : IDeploymentSource
     {
         private readonly ILayerService _layerService;
         private readonly ISiteService _siteService;
-        private readonly JsonSerializerOptions _serializationOptions;
+        private readonly JsonSerializerOptions _jsonSerializerOptions;
 
         public AllLayersDeploymentSource(
             ILayerService layerService,
             ISiteService siteService,
-            IOptions<JsonDerivedTypesOptions> derivedTypesOptions)
```

Comment:
Why?

Reply:
because it is part of the `IOptions<JsonSerializerOptions>`